src/AWS.Deploy.Recipes/CdkTemplates/ConsoleAppECSFargateService/ConsoleAppECSFargateService.csproj

```diff
@@ -44,7 +43,7 @@
   </ItemGroup>
   <!-- The project has run through the project template engine and "AWSDeployRecipesCDKCommonVersion" has been replaced with valid version number so a PackageReference is used. -->
   <ItemGroup Condition=" '$(RecipeCDKCommonVersion.ToUpper())' != 'AWSDEPLOYRECIPESCDKCOMMONVERSION' ">
-    <PackageReference Include="AWS.Deploy.Recipes.CDK.Common" Version="AWSDeployRecipesCDKCommonVersion"/>
+    <PackageReference Include="AWS.Deploy.Recipes.CDK.Common" Version="AWSDeployRecipesCDKCommonVersion" />
```

Comment:
nit: undo whitespace change

Reply:
Actually I think the space is the default formatting VS does. So I think we should make this change so VS won't keep changing the value.